src/DependencyInjection/Compiler/DoctrineOrmMappingsPass.php

```diff
@@ -107,11 +107,11 @@ protected function getChainDriverServiceName(ContainerBuilder $container)
      *                                       enable the mapping. Set to false to not do any check,
      *                                       optional
      */
-    public static function createXmlMappingDriver(array $mappings, array $managerParameters = array(), $enabledParameter = false)
+    public static function createXmlMappingDriver(array $mappings, array $managerParameters = [], $enabledParameter = false)
     {
-        $arguments = array($mappings, '.orm.xml');
+        $arguments = [$mappings, '.orm.xml'];
         $locator = new Definition('Doctrine\Common\Persistence\Mapping\Driver\SymfonyFileLocator', $arguments);
-        $driver = new Definition('Doctrine\ORM\Mapping\Driver\XmlDriver', array($locator));
+        $driver = new Definition('Doctrine\ORM\Mapping\Driver\XmlDriver', [$locator]);
```

Comment:
you are right. but actually we don't need that class anymore at all. all doctrine bundle versions that are maintained contain this mappings pass, so we need no copy in core bundle. removing it, and adjusting the other bundles.